db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_10_06_003829) do
+ActiveRecord::Schema.define(version: 2021_02_16_010033) do
```

Comment:
should we track schema.rb? or should we let migration update schema?

Reply:
I think having schema.rb tracked is fine. It's a good idea to run migration beforehand in production and development.